### DISCUSSION TITLE
添加 领取礼物盒功能

### DISCUSSION THREAD

**Discussion by DNNDHH** [DNNDHH/F-D-L#2] [Announcements]
每次登录时，将自动领取 礼物盒中的（ QP 圣晶石 呼符 圣杯 方块 叶子 碎石头 ）

<hr /><em>This discussion was created from the release <a href='https://github.com/DNNDHH/F-D-L/releases/tag/v1.0.5'>添加 领取礼物盒功能 </a>.</em>

**Comment by DNNDHH** [comment]
由于原作者已经锁库，放弃项目并停止更新！ 留下的坑 我补完了。。。接下来就是日常维护而已。。。还需要什么新功能等我想到再加上

**Reply by idearstar** [reply]
大佬，我还想要过章的

**Reply by DNNDHH** [reply]
这个有现成的 https://github.com/hexstr/FateBot
